documentation/how-to/tracking.en.md

```diff
@@ -5,89 +5,91 @@ tx_slug: documentation_how-to_tracking
 
-:material-tablet: Fieldwork
+## Set-Up Tracking Session
 
-To activate tracking, you must first make sure the positioning is active. Then,
-open the side dashboard, long-press on a legend layer within which you want to
-save your tracks to, and select the *Setup tracking* button to configure
-the tracking session.
+1. Open the legend and press long on the layer which you want to use for tracking
+2. Select **Setup tracking** and a new window will appear showing the configuration settings
 
-!![](../assets/images/tracking-layer-properties.png)
+    !![](../assets/images/tracking-layer-properties.png,350px)
 
-Two constraints are available to limit the number of vertices recorded
-during tracking:
-- A minimum time interval
-- A minimum distance
+    You can choose between two options how the number of vertices will be recorded during the tracking session:
 
-!![](../assets/images/tracking-settings.png)
+    - The `Time` tolerance records your position at a regular rate (e.g., every 30 seconds), which is ideal for consistent logs and saving battery.
+    - The `Distance` tolerance is more efficient, recording a point only after you move a minimum distance.
+    This keeps your track clean by ignoring movement when you are stationary.
+    To capture fine detail on steep or winding terrain, use a shorter distance, for straight routes, a longer distance is better.
 
-Once you have finished configuring your tracking session, hit the *Start tracking*
-button to begin recording. At this stage, a feature form will appear, which allows
-you to define the attributes of the feature(s) that will be created while QField
-tracks your position.
+    !![](../assets/images/tracking-settings.png,350px)
 
-For line and polygon layers, a single feature - its geometry formed of vertices
-from the recorded positions - will be create per tracking session.  For point
-layers, a feature will be created for each recorded position, with attribute
-values entered in the form remembered across features.
+    In the case that you wish to ensure that the next vertex is set not too far away from the most recent one (for example if you are in steep terrain and elevation is relevant), you can set a maximum tolerance distance.
 
-To prevent overlapping, you have the option to set a maximum distance tolerance between the last and next tracked point/vertices. This ensures that if the position exceeds this defined distance, it is considered as an erroneous entry.
+    !![](../assets/images/maximum-distance-tolerance.png,350px)
 
-!!! note
-    QField will skip the feature form step if the layer is configured to hide all attributes or if form has been set to hide on feature addition in the attribute form settings. This can streamline the process of starting tracking sessions.
+3. Tap the **Start tracking** button to begin the tracking.
+4. Set the attributes for the feature(s) about to be created.
```

Comment:
See here the bullet point presentation definitively feels off, it's intertwined with extra paragraphs and images.